lib/src/main/java/io/ably/lib/http/Http.java

```diff
@@ -81,21 +77,14 @@
 	 *     Public API
 	 *************************/
 
-	public Http(AblyRest ably, ClientOptions options) {
-		this.ably = ably;
+	public Http(ClientOptions options, Auth auth) {
+		this.options = options;
+		this.auth = auth;
+		this.activeHost = options.restHost;
 		this.scheme = options.tls ? "https://" : "http://";
 		this.port = Defaults.getPort(options);
 	}
 
-	private String getPrefHost() {
```

Comment:
The reason for this was that we would prefer to use the same host for REST requests as we are connected to, in the case of a realtime library. I would prefer that we retain this, because if it uses a different fallback host it is likely to be in a different region, which causes problems for certain requests.